### ISSUE TITLE
Regarding internship table structure

### ISSUE THREAD

**Issue by GintasS** [jenndryden/Canadian-Tech-Internships-Summer-2022#13]
I would suggest making the list same as the one for US market, https://github.com/Pitt-CSC/Summer2021-Internships, as you mentioned in the readme itself that you took some inspiration from it, I would highly suggest making one global table, instead of sorting entries by opening/closing times of the application, because:

-  Dates are very volatile and they can constantly change, so organizing internships around it is not a good idea.
- Keeping up with the applications will be hard, and you making your life even harder by adding a date relationship to this idea.

TL DR: make table as one, instead of organizing it around open/close times of the applications.

**Comment by GintasS** [comment]
@ChrisDryden how about this, did you have time to think about it?

**Comment by danber8** [comment]
M6.   ok
